atox/src/main/kotlin/ui/user_profile/UserProfileFragment.kt

```diff
@@ -72,7 +86,6 @@ class UserProfileFragment : BaseFragment<FragmentUserProfileBinding>(FragmentUse
 
         userToxId.text = vm.toxId.string()
 
-        // TODO(robinlinden): This should open a nice dialog where you show the QR and have both share and copy buttons.
```

Comment:
Leave the comment. :P I was thinking that tapping the ToxID should display the QR code and have share/copy buttons for the ToxID. Right now it's not obvious how to get to the QR code.

Reply:
OK. But I will remove it when we merge the UX/UI redesign.